### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by josenavas**
No changes made, just moved the code from the qiita repo.

### CODE REVIEW

File: tgp/pick_otus.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    job_id : str
+        The job id
+    parameters : dict
+        The parameter values to run split libraries
+    out_dir : str
+        Yhe path to the job's output directory
+
+    Returns
+    -------
+    dict
```
Comment (mortonjt) [inline tgp/pick_otus.py]: Is this out dated?

File: tgp/split_libraries/split_libraries.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+        The job id
+    parameters : dict
+        The parameter values to run split libraries
+    out_dir : str
+        Yhe path to the job's output directory
+
+    Returns
+    -------
+    dict
+        The results of the job
```
Comment (mortonjt) [inline tgp/split_libraries/split_libraries.py]: same here

**Comment by mortonjt** [comment]
First pass through.  Just minor documentation issues.  I think it should be ok.

**Comment by antgonza** [comment]
Are you planning to add travis?

**Comment by josenavas** [comment]
Not in this PR - since I've to modify the tests to remove httpretty I will enable then

**Pull request merged**